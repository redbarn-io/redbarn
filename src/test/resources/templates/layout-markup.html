
<!-- Supply page specific stylesheets links -->
<link type="redbarn/css" href="foo.css" rel="stylesheet">
<link type="redbarn/css" href="bar.css" rel="stylesheet">

<!-- Supply page specific inline styles -->
<style id="page-styles">
    .foo {
        background-color: transparent;
    }
</style>

<!-- Page content -->
<section id="content">
    <ul class="fruit">
        <li>foo</li>
    </ul>
    <p class="meat">
        Bacon ipsum dolor amet short ribs pork belly rump, biltong jowl fatback
        strip steak. Beef ribs spare ribs pork loin pork belly boudin landjaeger
        salami biltong pork kevin. Tri-tip sausage kielbasa flank. T-bone
        fatback kielbasa boudin salami corned beef. Pig ball tip fatback,
        biltong flank beef kevin picanha turkey leberkas venison short ribs
        shank. Brisket rump chuck, tail salami burgdoggen short ribs leberkas.
        Kielbasa fatback tri-tip porchetta, meatloaf shankle tenderloin short
        ribs turkey.
    </p>
    <p class="meat">
        Pork loin kevin cow pig shoulder. Short ribs andouille kevin pork loin
        turkey strip steak capicola bresaola tongue spare ribs turducken tail
        t-bone chuck. Meatloaf bacon tongue, pig pancetta shankle ball tip
        sausage shoulder pastrami tri-tip short ribs. Pork chop tail pork belly
        pork loin swine chuck. Hamburger pancetta spare ribs short ribs jowl,
        jerky kevin corned beef drumstick picanha shankle. Pig leberkas
        tenderloin, cow jowl sausage biltong swine. Tail sirloin rump, ham hock
        chicken pork chop cupim fatback sausage boudin flank hamburger jerky.
    </p>
</section>

<!-- Page Script "Sources" -->
<section id="page-script-src">
    <script type="text/javascript" src="foo.js"></script>
    <script type="text/javascript" src="bar.js"></script>
    <script type="text/javascript" src="baz.js"></script>
</section>

<!-- Supply page specific inline scripts -->
<script id="page-scripts">
    function foo() {
        console.log('bar');
    }
</script>

