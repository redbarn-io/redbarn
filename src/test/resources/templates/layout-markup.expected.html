<!doctype html>
<html lang="en">
    <head>
      <meta charset="UTF-8">
      <title>Foo Bar</title>
      <link href="vendor.css" rel="stylesheet">
      <link href="app.css" rel="stylesheet">
      <link href="foo.css" rel="stylesheet">
      <link href="bar.css" rel="stylesheet">
      <style id="page-styles">
          .foo {
              background-color: transparent;
          }
      </style>
    </head>
    <body>
    <section id="content">
        <ul class="fruit">
            <li>apples</li>
            <li>pears</li>
            <li>plums</li>
        </ul>
        <p class="meat">lorem ipsum dolar at sim.</p>
        <p class="meat">lorem ipsum dolar at sim.</p>
    </section>
    <footer>
        Copyright 2017
    </footer>
    <section id="page-script-src">
        <script type="text/javascript" src="foo.js"></script>
        <script type="text/javascript" src="bar.js"></script>
        <script type="text/javascript" src="baz.js"></script>
    </section>
    <script id="page-scripts">
        function foo() {
            console.log('bar');
        }
    </script>
    </body>
</html>